README.rst

```diff
@@ -69,9 +69,9 @@ Command Completion
 
 The aws-cli package includes a very useful command completion feature.
 This feature is not automatically installed so you need to configure it manually.
-To enable tab completion for bash use the built-in command ``complete``::
 
-    $ complete -C aws_completer aws
+For bash please refer to bin/aws_bash_completer.  Add that file under /etc/bash_completion.d,
```

Comment:
I would prefer to have the documentation for both approaches.  We can have users prefer the bash_competion.d script, but we should still document using `complete` directly, as it's still a valid way to enable this.